app/models/publisher.rb

```diff
@@ -103,6 +110,31 @@ class Publisher < ApplicationRecord
 
   store_accessor :feature_flags, VALID_FEATURE_FLAGS
 
+  def self.valid_payable_uphold_creators
+    where(selected_wallet_provider_type: UPHOLD_CONNECTION)
+    where('uphold_connections.country IS NULL').
+      or(
+        where(selected_wallet_provider_type: UPHOLD_CONNECTION).
+        where.not("uphold_connections.country ILIKE '#{UpholdConnection::JAPAN}'")
+      ).
+      joins(:uphold_for_join)
```

Comment:
```suggestion
      joins(:uphold_for_join).
      where.not("uphold_connections.country ILIKE '#{UpholdConnection::JAPAN}'")
```

I don't think you'd need to do `(country == null || country != "JP")`

Reply:
If you just say country != JP then it won't pick up NULL 
https://stackoverflow.com/questions/8036691/sql-not-displaying-null-values-on-a-not-equals-query

Reply:
Oh good point. I don't want to hold up the PR but the joins part at the bottom and the naming of the `belongs_to` kind of violates principle of least surprise. I think a scope that does a join with `ON publishers.selected_wallet_provider_id = uphold_connections.id AND publishers.selected_provider_type_provider = 'UpholdConnection'` would save on readability.

Reply:
Ah yes, I should clarify that AR complains if I put it at the top, but b/c of the way AREL works wants it at the bottom. I agree it looks better at the top.